src/main/java/ch/njol/skript/events/EvtServerTime.java

```diff
@@ -0,0 +1,147 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.SkriptEventHandler;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Time;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+
+public class EvtServerTime extends SkriptEvent {
+
+	public static class ServerTimeEvent extends Event {
+		@Override
+		public @NotNull HandlerList getHandlers() {
+			throw new IllegalStateException();
+		}
+	}
+
+	private static final long HOUR_12_MILLISECONDS = 43200000;
```

Comment:
```suggestion
	private static final long HOUR_12_MILLISECONDS = 12 * 60 * 60 * 1000;
```
will be replaced at compile time anyway, might as well make the math clear